tools/umi_tools/main.nf

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env nextflow
+
+// Specify DSL2
+nextflow.preview.dsl = 2
+
+// Local default params
+params.internal_process_name = 'umitools_dedup'
+
+// Process definition
+process umitools_dedup {
+    publishDir "${params.outdir}/${params.internal_process_name}",
+        mode: "copy", overwrite: true
+
+    container 'luslab/nf-modules-umitools:latest'
+
+    input:
+      tuple val(sample_id), path(bai), path(bam)
```

Comment:
```suggestion
      tuple val(sample_id), path(bam)
```

Reply:
As mentioned below, I think we should try and be consistent with staging the `.bai` files in the same channel elements as the `.bam` files. I cant think of any reason this would break anything and is much tidier.